cross/syncthing/Makefile

```diff
@@ -38,7 +38,7 @@ $(error Unsupported ARCH $(ARCH))
 endif
 
 ifeq ($(findstring $(SYNCTHING_ARCH),amd64),$(SYNCTHING_ARCH))
-SYNCTHING_BIN = $(WORK_DIR)/$(PKG_DIR)/bin/linux_amd64/syncthing
+SYNCTHING_BIN = $(WORK_DIR)/$(PKG_DIR)/bin/syncthing
```

Comment:
hrmm ok, I will fix it. I did test on `armada370` and `cedarview` as well. Do you have another favorite to use as a canary build?